pages/test_analytics/test_ownership.md

```diff
@@ -0,0 +1,117 @@
+# Test ownership
+Test ownership is critical to adopting a healthy testing culture at your organization. By defining test owners, your teams will become accountable for maintaining a fast and reliable test suite, ensuring confidence when you deploy your code.
+
+Customers on the [Pro and Enterprise plans](https://buildkite.com/pricing) can assign test ownership to [Teams](/docs/team-management/permissions).
+
+Test ownership is managed via team assignments in a TESTOWNER file. The team that is the default owner of a test [will be automatically assigned flaky tests](/docs/test-analytics/flaky-test-assignment) to triage.
+
+> 🚧 Buildkite test ownership is currently in private beta
+> Please reach out to our support team to register for early access.
+
+## Setting test ownership
+
+You can upload a TESTOWNER file via this API endpoint:
+
+```bash
+curl --location 'https://analytics-api.buildkite.com/v1/test-ownerships' \
+     --header "Authorization: Bearer <your-suite-api-token>" \
+     -F 'file=@<your-TESTOWNER-file-location>'
+```
+You might consider creating a new pipeline to automatically upload your TESTOWNER file when changes are detected.
+
+## Example TESTOWNER file
+
+```bash
+# This is a comment.
+# Each line is a file pattern followed by one or more team slugs.
+
+# These teams will be the owners for everything in
+# the repo. While both teams own the test, the first team listed will
+# become the default owner for tests in this directory. Unless
+# a later match takes precedence, team-slug-1 will have any
+# flaky tests assigned to them by default.
+*       team-slug-1 team-slug-2
+
+# Order is important; the last matching pattern takes the most
+# precedence. Any test file ending with _spec.rb will be assigned
+# to the test-analytics team and not team-slug-1.
+*_spec.rb    test-analytics #This is an inline comment.
+
+# Only Buildkite teams can be specified as test owners. Teams should
```

Comment:
This line should probably be moved to after `# This is a comment.` above.
Then the 2nd sentence can be deleted as it's already covered by the following line.